tests/keras/losses_test.py

```diff
@@ -109,6 +109,22 @@ def test_sparse_categorical_crossentropy_4d():
     assert np.isclose(expected_loss, np.mean(loss))
 
 
+def test_multi_hot_sparse_categorical_crossentropy():
```

Comment:
please add test for multi_hot_sparse_categorical_accuracy

Reply:
added, see changes at tests/keras/metrics_test.py